src/Og.php

```diff
@@ -35,30 +35,44 @@ class Og {
    *   The entity type.
    * @param $bundle
    *   The bundle name.
+   * @param $settings
+   *   (Optional) allow overriding the default definition of the field/instance.
```

Comment:
`instance` is no longer the right term, we should adapt function accordingly (field-storage, and field-config)